certificate-authority/pb/signingRecords_test.go

```diff
@@ -0,0 +1,189 @@
+package pb_test
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/plgd-dev/hub/v2/certificate-authority/pb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCredentialStatusValidate(t *testing.T) {
+	tests := []struct {
+		name    string
+		input   *pb.CredentialStatus
+		wantErr bool
+	}{
+		{
+			name: "Valid credential",
+			input: &pb.CredentialStatus{
+				Date:           1659462400000000000,
```

Comment:
this is `ValidFromDate` or it is date when record has been created/updated ?

Reply:
The second, it is the record itself as been created or updated